### DISCUSSION TITLE
Using openPGP keys to encrypt emails

### DISCUSSION THREAD

**Discussion by DimitrijeDimitrijevic** [the-djmaze/snappymail#1720] [Q&A] [answered]
Hello, I have generating the keys in webmail interface but here is the issue I have. 

When I want to send messages and to be encrypted I do not get any prompt to enter password, and messages sent is received unencrypted by other side. 

In application.ini file the openpgp option is enabled. 

What settings should I check ? 
What should I do to encrypt the message ? 

Thanks for you help!

**Comment by the-djmaze** [comment, marked as answer]
You must see these buttons to get it working (and green they are active):
![afbeelding](https://github.com/user-attachments/assets/e40b60fa-ef34-4161-87fa-a83f1605a82c)

This only happens when all email adresses have an encryption key.
Ask the recipients for their public key and import them.

**Reply by DimitrijeDimitrijevic** [reply]
Hi @the-djmaze , 

Thanks for answer, I will try that out for sure!

**Reply by DimitrijeDimitrijevic** [reply]
@the-djmaze Hello again, 

Here is the problem, I get sign icon but not encrypt one, I have imported public key from the address I am sending to, but I cannot encrypt it. 
![image](https://github.com/user-attachments/assets/31d51bc1-6c58-4684-8bd6-5f6fe7aec705)

This is under OpenPGP.js menu.

Please advise what to do next.

Thanks!

**Reply by DimitrijeDimitrijevic** [reply]
Hi @the-djmaze again, 

Sorry for not introducing myself here first. 
I am Dimitrije from https://migadu.com
We are using snappymail as our webmail client, version we are using is 2.35.0 on FreeBSD.
The issue is that encrypt button is not shown in compose email, it was hidden, until I enabled it trough JS console in Firefox. 
Was there any similiar issues with this version ?

**Reply by the-djmaze** [reply]
It could be. Try with latest release?
